systems/messaging/nns/pb/nns.proto

```diff
@@ -56,8 +53,8 @@ message SetNodeIPRequest{
     int32 nodePort = 4;
     int32 meshPort = 5;
```

Comment:
Please renumber this starting from 1 and regenerate it. I know it's not part of your changes, but if you could fix this as well, please do.

Reply:
idk whats you are referring here. Here's file link
https://github.com/ukama/ukama/blob/638b246ae779163c4d401ddb2a5f28bde73937db/systems/messaging/nns/pb/nns.proto